main_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"flag"
+	"github.com/Shopify/sarama"
+	"github.com/Shopify/sarama/mocks"
+	"github.com/companieshouse/chs.go/kafka/producer"
+	. "github.com/smartystreets/goconvey/convey"
+	"testing"
+)
+
+func TestUnitProcessMessages(t *testing.T) {
+	offsetArraySingle := []int64{1}
+
+	Convey("test successful - message published", t, func() {
+		consumerMock := mocks.NewConsumer(t, nil)
+		consumerGroup := consumerMock.ExpectConsumePartition("", 0, 1)
+		consumerGroup.YieldMessage(&sarama.ConsumerMessage{})
+
+		producerMock := mocks.NewSyncProducer(t, nil)
+		producerMock.ExpectSendMessageAndSucceed()
+		argu := Arguments{
+			OffsetArray: offsetArraySingle,
+			Consumer:    consumerMock,
+			Producer:    &producer.Producer{producerMock},
+		}
+		processMessages(argu)
+	})
+}
+
+func TestUnitValidateFlags(t *testing.T) {
+	flag.Set("broker", "broker")
+	flag.Set("schema", "schema")
+	flag.Set("offset", "1")
+	flag.Set("schema-registry", "schema-registry")
+	flag.Set("topic", "topic")
+
+	Convey("test successful", t, func() {
```

Comment:
The test description here should state what is being tested.

Reply:
description add in [9d973a2](https://github.com/companieshouse/kafka-management-tool/pull/1/commits/9d973a26f65842bc2413165487e5c744ede2612b)